content/post/wrong-data.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Randy Clinton
+title: "Wrong data can still be right"
+publishDate: "????"
+cover: "????"
+draft: true
+tags: 
+ - data-integrity
+ - communication
+ - errors
+---
+
+Let us explore what it means for data to be wrong. For now, think of it from the user’s reaction only. It is a data point, trend line, or report that makes the user lean into their screen, squint, tilt their head a bit, and say, “nooooo, that’s wrong.”  
+
+Wanting to jump in and “fix” it is a natural feeling, especially for people who like the technical side of data. But let me make the case to pause for a beat and think through what “wrong” could mean before you head off fixing something that is not actually broken. 
+<!--more-->
+I propose there are three common situations in which data is considered wrong, and only one of them would fit a literal definition of “wrong.” They are (in order from least to most common):
+
+<ul>
+<li> The data consumer is using a faulty source to “check” the data.
+<li> The data is literally wrong. 
```

Comment:
Propose striking "literally wrong" in favor of "there is a bug in the code" or "the source data are incorrect".